Rakefile

```diff
@@ -32,6 +32,21 @@ task :vendor_deps do
   sh 'bower install'
   sh 'cp', 'bower_components/test_track_js_client/dist/testTrack.bundle.min.js', 'app/assets/javascripts'
 
+  # Download testtrack-cli
+  TEST_TRACK_CLI_VERSION = 'v0.9.7'
+  FileUtils.module_eval do
+    mkdir_p 'vendor/bin/testtrack-cli'
+    cd 'vendor/bin/testtrack-cli' do
+      rm_r Dir.glob('*')
+
+      %w(darwin linux).each do |arch|
+        `curl -L https://github.com/Betterment/testtrack-cli/releases/download/#{TEST_TRACK_CLI_VERSION}/testtrack.#{arch} \
```

Comment:
One fun thing I learned is that this gets rate limited pretty readily when used unauthenticated. I spose it doesn't matter much for this operation, though, you only rarely need to do this, and can cope with some pain if it arises.